tla-io/src/main/scala/at/forsyte/apalache/io/quint/Quint.scala

```diff
@@ -331,6 +331,18 @@ class Quint(moduleData: QuintOutput) {
         case "item"   => binaryApp(opName, tla.app)
         case "tuples" => variadicApp(tla.times)
 
+        // Maps (functions)
+        // case "Map"       => null // Rewire with "Apalache!SetAsFun"
+        // case "Map"       => variadicApp(args => tla.appOp("Apalache!SetAsFun", tla.enumSet(args: _*)))
```

Comment:
Note that `Apalache!SetAsFun` is a built-in operator, so a translation similar to that of `TlaSetOper!enumSet`.